demo/admin/src/App.tsx

```diff
@@ -182,7 +182,21 @@ class App extends React.Component {
                                                                                     />
                                                                                     <RouteWithErrorBoundary
                                                                                         path={`${match.path}/assets`}
-                                                                                        render={() => <DamPage path="/assets" />}
+                                                                                        render={() => (
+                                                                                            <DamPage
```

Comment:
doing all this in App is a bit much, probably move it into it's onw Page component?

Reply:
I'd rather move the Content Scope rendering into its own file, as we may reuse it for other features that are scope only by domain. What do you think?

Reply:
Done: 666985a52bd4a3119698df217cb6d69143bac886